Helper/Order.php

```diff
@@ -287,7 +287,7 @@ public function fetchTransactionInfo($reference, $storeId = null)
      *
      * @throws \Exception
      */
-    private function setShippingMethod($quote, $transaction)
```

Comment:
What is the reasoning to set all these `protected`? Just for easier testing convenience?

Reply:
Yep. Some were complicated to test as private.